next.mdx.use.mjs

```diff
@@ -66,8 +66,8 @@ export const mdxComponents = {
     BitnessDropdown: BitnessDropdown,
     // Renders a drop-down menu from which the operating system can select
     OperatingSystemDropdown: OperatingSystemDropdown,
-    // Renders a npm version of the selected release
-    NpmVersion: NpmVersion,
+    // Renders a link to the NPM version of the selected release
```

Comment:
`npm` should never be capitalized https://github.com/npm/cli?tab=readme-ov-file#faq-on-branding

```suggestion
    // Renders a link to the npm version of the selected release
```